finetune/base_models/gpt2/encoder.py

```diff
@@ -51,6 +51,21 @@ class GPT2Encoder(BaseEncoder):
 
     def __init__(self, encoder_path=ENCODER_PATH, vocab_path=VOCAB_PATH):
         super().__init__(encoder_path=encoder_path, vocab_path=vocab_path)
+        self.freqs = {}
+        index = 0
+        with open("dict.txt", "r", encoding="utf-8") as freq_dict:
```

Comment:
I only see mention of dict.txt for bert and roberta. Is there a separate one for GPT-2? And it'll be in the gpt2 directory?

Reply:
Correct.  Slightly different name but same idea.

Reply:
Dict.txt is not needed for gpt2 - sorry, will remove